### ISSUE TITLE
Running cycle

### ISSUE THREAD

**Issue by moxerox** [abdullahkahled/charge#2]
The code currently runs every 30 seconds, which consumes a lot of unnecessary resources and the charge is less likely to drop in a 30 seconds difference.

My suggestion is to change the loop to run once every 5 or 10 minutes.

The most optimum solution in this case is to leave it up to each user, so we can use 2 variables to decide how often the code should run, and the percentage of when the notification should fire.

**Comment by abdullahkahled** [comment]
Certainly, the user is the one who can set the time, but you are correct. The time between each verification and the other must be at least two or three minutes. This is correct.
Also, the percentage we can make it up to 90 is okay, and this is up to the user, as you kindly do

Thank you for that note
